src/matches.ts

```diff
@@ -1,5 +1,10 @@
 export type NormalizerFn = (textToNormalize: string) => string;
+
 export type TextMatch = string | RegExp;
+export type TextMatchOptions = {
```

Comment:
did you move it in this file to make it closer to NormalizerFn?

Reply:
I've moved it primarily to put it ntext to `TextMatch` :-)